azext_iot/central/services/api_token.py

```diff
@@ -10,10 +10,13 @@
 from azext_iot.constants import CENTRAL_ENDPOINT
 from azext_iot.central.services import _utility
 from azext_iot.central.models.enum import Role
+from azure.cli.core.util import should_disable_connection_verify
 
 logger = get_logger(__name__)
 
 BASE_PATH = "api/preview/apiTokens"
+http = requests.Session()
```

Comment:
Do we need to make similar changes to how the `monitor-events` work? 

https://github.com/Azure/azure-iot-cli-extension/blob/a5e3accbb036511e618114a988ce19f8af22c9db/azext_iot/common/_azure.py#L122

cc: @digimaun

Reply:
I think we should be consistent and that function should be at play due to the `requests.post` against a central app endpoint. Good catch.